blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,81 @@
+/* !important added to override system styles to maintain cutoff 
+effect on filter buttons to indicate horizontal scroll */
+.filter-group-wrapper {
+  padding-right: 0 !important;
+
+  @media (min-width: 76rem) {
+    padding: 0 2rem !important;
+  }
+}
+
+.filter-group {
+  margin-block: 1.5rem;
+}
+
+.filter-group__list {
+  display: flex;
+  gap: 0.5rem;
+  overflow-x: auto;
+  padding-block-end: 1rem;
+  padding-inline-start: 0;
+  scrollbar-width: none; /* Firefox */
+  border-bottom: 1px solid var(--spectrum-gray-700);
+  transition: padding-inline-start 100ms ease-in-out;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.filter-group__list::-webkit-scrollbar {
+  display: none;
+}
+
+.filter-group__list:has(.filter-group__button:first-child:focus) {
+  padding-inline-start: 0.5rem;
+}
+
+.filter-group__label {
+  margin-block-end: 0.5rem;
+  color: var(--spectrum-gray-700);
+  font-size: 1rem;
+}
+
+.filter-group__button {
+  --background-color: light-dark(
```

Comment:
The `--background-color` variable could be meant for a page or element background, not a button. Let's use a more specific custom property here, and follow our macro -> micro naming convention as set out in the functional variables section of `base.css` 

```css
--color-context-specifics

--color-background-button-filters-default 
--color-background-button-filters-hover

// etc
```

Reply:
more clear for people who need to edit code later and prevents collisions